components/Marketing/MiniFront.js

```diff
@@ -9,14 +9,16 @@ const MiniFront = ({ data: { loading, error, front }, t }) => {
     <Loader
       loading={loading}
       error={error}
-      render={() => (
-        <RenderFront
-          t={t}
-          isEditor={false}
-          front={front}
-          nodes={front.children.nodes}
-        />
-      )}
+      render={() =>
+        front ? (
+          <RenderFront
+            t={t}
+            isEditor={false}
+            front={front}
+            nodes={front.children.nodes}
+          />
+        ) : null
```

Comment:
was this an issue that happened often? or more of a dev edge case? maybe also a darn apollo bug, can stay in if convenient in dev otherwise this might mask a real problem of the marketing front not properly loading.

Reply:
dev edge case... it could theoretically cause problems if there is no marketing front document defined.